open-banking-accelerator/accelerators/ob-apim/repository/conf/configure.properties

```diff
@@ -0,0 +1,41 @@
+ # Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ #
+ # WSO2 LLC. licenses this file to you under the Apache License,
+ # Version 2.0 (the "License"); you may not use this file except
+ # in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ #    http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing,
+ # software distributed under the License is distributed on an
+ # "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ # KIND, either express or implied. See the License for the
+ # specific language governing permissions and limitations
+ # under the License.
+
+# Specify the hostname you want to configure
+APIM_HOSTNAME=localhost
+IS_HOSTNAME=localhost
+BI_HOSTNAME=localhost
+
+# Specify the deployment.toml file containing the required configurations based on the APIM version being used
+# As an example if APIM 4.1.0 version is used change the file name to wso2am-4.1.0-deployment.toml in the same file path
+PRODUCT_CONF_PATH=repository/resources/wso2am-4.0.0-deployment.toml
```

Comment:
Shall we have the latest supported base product deployment.toml version here?

Reply:
fixed with 7e28df1b5143cc76698694a7bdf3e2403561c61d